Project.toml

```diff
@@ -29,6 +29,7 @@ PrecompileTools = "aea7be01-6a6a-4083-8856-8a6e6704d82a"
 Preferences = "21216c6a-2e73-6563-6e65-726566657250"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
+RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
```

Comment:
Is this used in Trixi.jl?

Reply:
Not directly, but it is an indirect dependency through StartUpDG. I added it explicitly so I could add a compat bound.

Reply:
The compat bound is to avoid this issue https://github.com/trixi-framework/Trixi.jl/issues/2061; resolving it should allow us to remove RecursiveArrayTools as an explicit dependency as well. 

Adding a note of this to #2061.

Reply:
Thanks!